sandbox/modules/identity/src/main/java/org/opensearch/identity/User.java

```diff
@@ -71,6 +76,16 @@ public void setBcryptHash(String bcryptHash) {
         this.bcryptHash = bcryptHash;
     }
 
+    @JsonProperty(value = "permissions")
+    public List<String> getPermissions() {
+        return permissions;
+    }
+
+    @JsonProperty(value = "hash")
```

Comment:
Could you explain the choice to make this be the "hash" element?

Reply:
Good catch, that should read `permissions`

Reply:
Updated